cms/templates/js/license-selector.underscore

```diff
@@ -0,0 +1,113 @@
+<div class="license-wrapper">
+    <h3 class="label setting-label">
+        <%= gettext("License Type") %>
+    </h3>
+    <ul class="license-types">
+        <% var link_start_tpl = '<a href="{url}" target="_blank">'; %>
+        <% _.each(licenseInfo, function(license, licenseType) { %>
+            <li data-license="<%- licenseType %>">
+                <button name="license-<%- licenseType %>"
+                        class="action license-button <% if(model.type === licenseType) { print("is-selected"); } %>"
```

Comment:
@singingwolfboy, the state of the button here is an important part of the context for screen reader users, too since it is the only way they can tell what license type is selected.  That is clear to a sighted user thanks to the is-selected style which styles it blue, but classes have no effect on what gets read to the screen reader.  You can solve that by toggling the value of ```aria-pressed``` true/false http://www.w3.org/TR/wai-aria/states_and_properties#aria-pressed